src/routes/chains/converters.rs

```diff
@@ -36,6 +36,16 @@ impl From<BackendChainInfo> for ServiceChainInfo {
                 },
                 value: chain_info.safe_apps_rpc_uri.value,
             },
+            public_rpc_uri: ServiceRpcUri {
+                authentication: match chain_info.public_rpc_uri.authentication {
+                    RpcAuthentication::ApiKeyPath => ServiceRpcAuthentication::ApiKeyPath,
+                    RpcAuthentication::NoAuthentication => {
+                        ServiceRpcAuthentication::NoAuthentication
+                    }
+                    RpcAuthentication::Unknown => ServiceRpcAuthentication::Unknown,
+                },
+                value: chain_info.public_rpc_uri.value,
+            },
```

Comment:
I am starting to thing that we could add an implementation of `impl From<RpcUri> for ServiceRpcUri`. Wdyt?

Reply:
Yeah good point. Added in 4fce4f5